src/partitioned-dbs/index.rst

```diff
@@ -105,14 +105,20 @@ sensor readings from an IoT field monitoring service. In this particular
 use case, it's quite logical to group all documents by their ``sensor_id``
 field. In this case, we would call the ``sensor_id`` the partition.
 
+A partition offers two benefits. Firstly, it allows you to simplify the
+workload on the query engine, by allowing the engine to only query shards
+that it needs to query. Secondly, it allows the database to scale in more
+predictable ways.
+
 A good partition has two basic properties. First, it should have a high
 cardinality. That is, a large partitioned database should have many more
-partitions than documents in any single partition. A database that has
-a single partition would be an anti-pattern for this feature. Secondly,
-the amount of data per partition should be "small". The general
-recommendation is to limit individual partitions to less than ten
-gigabytes (10 GB) of data. Which, for the example of sensor documents,
-equates to roughly 60,000 years of data.
+partitions than documents, compared to an unpartitioned database. A database
+that has a single partition would be an anti-pattern for this feature, as
+one of the major bonuses of partitioning is that it allows index lookups to
+iterate over a smaller B-Tree. Secondly, the amount of data per partition
+should be "small". The general recommendation is to limit individual
+partitions to less than ten gigabytes (10 GB) of data. Which, for the example
```

Comment:
What is this recommendation based on?

Would it not make more sense to make a recommendation based on index size than on data size? Given that index size isn't trivial to determine, maybe basing this recommendation on number of documents (which is usually a closer proxy to index size) would make sense?

@wohali probably has better insight into the technical details here than me.

Reply:
https://blog.cloudant.com/2019/03/05/Partition-Databases-Introduction.html has an excellent introduction, see under _What makes a good partition key?_

We can't cut and paste that text, but it should help.

Note it's a 4-part blog series; it'd be good to read all of it before moving ahead.